ui/raidboss/data/04-sb/trial/suzaku.ts

```diff
@@ -45,23 +44,29 @@ const triggerSet: TriggerSet<Data> = {
     {
       id: 'Suzaku Southron Star',
       type: 'StartsUsing',
-      netRegex: NetRegexes.startsUsing({ id: '3234', source: 'Suzaku', capture: false }),
+      netRegex: { id: '3234', source: 'Suzaku', capture: false },
       response: Responses.aoe(),
     },
     {
+      // TODO: HeadMarker trigger
       id: 'Suzaku Phantom Flurry',
-      netRegex: NetRegexes.startsUsing({ id: '3231', source: 'Suzaku' }),
+      type: 'HeadMarker',
+      // netRegex: { id: '3231', source: 'Suzaku' },
+      netRegex: { id: '3231' },
```

Comment:
?

Reply:
Oh, I mis read the line here....

Reply:
I meed to add type in this trigger and misread to the next headmarker trigger ...